src/croniter/tests/test_croniter_hash.py

```diff
@@ -146,7 +146,7 @@ def test_hash_word_midnight(self):
 
         @midnight is actually up to 3 hours after midnight, not exactly midnight
         """
-        self._test_iter('@midnight', datetime(2020, 1, 1, 1, 10, 32), timedelta(days=1))
+        self._test_iter('@midnight', datetime(2020, 1, 1, 2, 10, 32), timedelta(days=1))
```

Comment:
this should be the behavior, if this test fails with new code, you introduced a bug

Reply:
This change comes from the previous PR https://github.com/kiorky/croniter/pull/42/files#diff-12057f09bdfcae88f7440dc7b5fbe65ee63141a58faec23c2fb54662caf3111fL147.
If I'm correct, it just fixes it

Reply:
still this one :)